utils/fixed_decimal/src/decimal.rs

```diff
@@ -3155,6 +2508,13 @@ fn test_from_str() {
         assert_eq!(fd.nonzero_magnitude_end(), cas.magnitudes[2], "{cas:?}");
         let input_str_roundtrip = fd.to_string();
         let output_str = cas.output_str.unwrap_or(cas.input_str);
+
+        // Check if the output string starts with a plus sign and remove it,
+        // because we do not show sign for unsigned fixed decimal
```

Comment:
Suggestion: maybe just reject strings with a `+` sign in UnsignedRoundingMode. It's okay to be more strict.